### PR TITLE
[VIEW API] App insights graph data

### PR DESCRIPTION
**Pull request by NoahNsimbe**
**_WHAT DOES THIS PR DO?_**

- [x] Modifies the view api to fetch data from BigQuery
- [x] Changes the format of data returned by the API to match the format required by the app in order to reduce the  extra work done to format data while on mobile phones.

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - []()
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**
```
./mvnw spring-boot:run -Dspring.profiles.active=api
```

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

File: .github/workflows/deploy-apis-to-production.yml [hunk 1]
```diff
@@ -1193,6 +1193,8 @@ jobs:
           kubectl create configmap --dry-run=client -o yaml --from-env-file=api.env prod-view-api-config | kubectl replace -f - -n production
           gcloud secrets versions access latest --secret="prod-env-view-message-broker" > message_broker.env
           kubectl create configmap --dry-run=client -o yaml --from-env-file=message_broker.env prod-view-message-broker-config | kubectl replace -f - -n production
+          gcloud secrets versions access latest --secret="prod-key-view-api-google-credentials" > google_application_credentials.json
+          kubectl create configmap --dry-run=client -o yaml --from-file=google_application_credentials.json view-api-config-files | kubectl replace -f - -n production
```
Comment (NoahNsimbe) [inline .github/workflows/deploy-apis-to-production.yml:1197]: Configuring google credentials.

File: k8s/view-api/prod-view-api.yaml [hunk 2]
```diff
@@ -83,6 +83,13 @@ spec:
             preStop:
               exec:
                 command: ["sh", "-c", "sleep 10"]
+          volumeMounts:
+            - name: config-volume
+              mountPath: /etc/config
+      volumes:
+        - name: config-volume
+          configMap:
+            name: view-api-config-files
```
Comment (NoahNsimbe) [inline k8s/view-api/prod-view-api.yaml:92]: mount volumns for google credentials

File: src/view/src/main/java/airqo/services/BigQueryApiImpl.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+				Frequency.HOURLY, hourlyDataTable, siteId, simpleDateFormat.format(startDateTime),
+				simpleDateFormat.format(endDateTime));
+
+			String dailyDataQuery = String.format(
+				"SELECT '%s' as frequency, timestamp, site_id, pm10, pm2_5, false as forecast " +
+					"FROM `%s` " +
+					"WHERE site_id = '%s' and timestamp >= '%s' and timestamp <= '%s' " +
+					"and pm2_5 is not null and pm10 is not null and timestamp is not null",
+				Frequency.DAILY, dailyDataTable, siteId, simpleDateFormat.format(startDateTime),
+				simpleDateFormat.format(endDateTime));
```
Comment (NoahNsimbe) [inline src/view/src/main/java/airqo/services/BigQueryApiImpl.java:56]: Querying both historical hourly and daily data

**Review by 123MwanjeMike — APPROVED** [review]
Thanks @NoahNsimbe 
LGTM

**Review by Baalmart — APPROVED** [review]
LGTM, thanks @NoahNsimbe

**Pull request merged**